### PR TITLE
Run PagerDuty sync script on cronjob

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Best reviewed commit by commit.

The `Write PagerDuty sync cron job` commit should be viewed [with whitespace disabled](https://github.com/alphagov/govuk-rota-generator/pull/74/commits/49dead9d8a6d18aa35a00a33670bd4655b78620d?w=1), to minimise the diff, since there are few code changes apart from indentation.

Trello: https://trello.com/c/hGaawZt1/220-automate-pagerduty-on-a-cron

### CODE REVIEW

File: lib/google_sheet.rb [hunk 1]
```diff
@@ -14,8 +28,16 @@ def initialize(sheets_api: Google::Apis::SheetsV4::SheetsService.new, scope: :re
               Google::Apis::SheetsV4::AUTH_SPREADSHEETS
             end
 
+    json_key_io = if !GoogleSheet::GOOGLE_SERVICE_ACCOUNT_KEY.nil?
```
Comment (KludgeKML) [inline lib/google_sheet.rb:31]: will `if GoogleSheet::GOOGLE_SERVICE_ACCOUNT_KEY` not work here? (it's going to be either nil or truthy, right?)

File: README.md [hunk 2]
```diff
@@ -122,5 +121,5 @@ By default, the script will ask you to approve each override: `y` to override, `
 If you wish to approve all of the overrides at once, you can pass the `--bulk` option:
```
Comment (KludgeKML) [inline README.md:129]: Not sure from context if this example should or shouldn't have the --bulk option?

Comment (ChrisBAshton) [reply README.md]: Oops, good spot! Fixed.

File: lib/google_sheet.rb (hunk 1 above)
Comment (ChrisBAshton) [reply lib/google_sheet.rb]: Agreed 👍 removed and tested locally, works ok.

**Review by KludgeKML — APPROVED** [review]
LGTM!

**Pull request merged**